docs/getting-started-with-cosmosdb.md

```diff
@@ -0,0 +1,103 @@
+## Getting Started with Scalar DB and Cosmos DB
+
+## Overview
+This document briefly explains how you can get started a simple electronic money application with scalar DB and Cosmos DB.
+
+## Install prerequisites
+
+Scalar DB is written in Java and uses Cosmos DB as an underlining storage implementation, so the following software is required to run it.
+
+* [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (OpenJDK 8) or higher
+* Other libraries used from the above are automatically installed through gradle
+
+### Cosmos DB setup
+
+* [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction)
```

Comment:
I think it abuses the bullet points. Please explain.
Specifying the link can't make users understand your intention.

Reply:
Sorry for the inconvenience.
Document updated.